### PR TITLE
debug: improve mm debug output

### PR DESCRIPTION
**Pull request by jinliangli**
- Use printk instead of printf in mm critical region, otherwise, system
  will crash, specially in SMP system.

- Remove "0x" prefix before "%p", "0x" is redundant for "%p".

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 93bbe13e8ce85fe2c289c5da771cf57169481a81

**Comment by skylarCai** [comment]
auto build success,number:222

**Comment by skylarCai** [comment]
autotest fail,number:1629

**Review by yanxiaoyong-ch — COMMENTED** [review]
this look ok for me

**Comment by skylarCai** [comment]
start build

**Comment by skylarCai** [comment]
手动触发！
感谢 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
auto build success,number:225

**Comment by skylarCai** [comment]
autotest fail,number:1689

**Comment by skylarCai** [comment]
autotest fail,number:1690

File: kernel/rhino/k_mm_debug.c [hunk 1]
```diff
@@ -9,27 +9,6 @@
 #include "aos/debug.h"
```
Comment (skylarCai) [inline kernel/rhino/k_mm_debug.c:9]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**